R/extract.R

```diff
@@ -191,6 +191,18 @@ extract_preprocessor.workflow <- function(x, ...) {
   abort("The workflow does not have a preprocessor.")
 }
 
+#' @rdname extract-workflow
+#' @export
+extract_postprocessor.workflow <- function(x, ...) {
+  if (has_post(x)) {
+    # TODO: to mirror the other methods, this should actually live
```

Comment:
What's stopping it from living in `x$post$actions$container$container`? Maybe the need for the new object name?

Reply:
I think just a misspecification of a constructor argument somewhere in the PR. Haven't had time to troubleshoot that yet.